terraform/ecs.tf

```diff
@@ -104,6 +104,10 @@ resource "aws_ecs_task_definition" "webapp" {
       {
         name : "JWT_SECRET",
         valueFrom : aws_secretsmanager_secret.webapp_next_auth_jwt_secret.arn
+      },
+      {
+        name : "AUTH_SECRET",
+        valueFrom : aws_secretsmanager_secret.webapp_next_auth_jwt_secret.arn
```

Comment:
Do we still need this AND `JWT_SECRET`?

Reply:
im thinking belt and braces here. if ever the AUTH_SECRET becomes manditory it will need to be defined.
https://next-auth.js.org/configuration/options#nextauth_secret